rmw_zenoh_cpp/src/detail/graph_cache.cpp

```diff
@@ -84,12 +91,31 @@ void GraphCache::parse_put(const std::string & keyexpr)
       }
 
       const liveliness::TopicInfo topic_info = entity.topic_info().value();
-      GraphNode::TopicMap & topic_map = entity.type() ==
-        EntityType::Publisher ? graph_node.pubs_ : graph_node.subs_;
-      const std::string entity_desc = entity.type() ==
-        EntityType::Publisher ? "publisher" : "subscription";
-      const std::size_t pub_count = entity.type() == EntityType::Publisher ? 1 : 0;
+      std::string entity_desc = "";
+      GraphNode::TopicMap & topic_map =
+        [&]() -> GraphNode::TopicMap &
```

Comment:
We really only need to capture `entity_desc`, right?

```suggestion
        [&entity_desc]() -> GraphNode::TopicMap &
```

Reply:
Done https://github.com/ros2/rmw_zenoh/pull/90/commits/aae76cdfd1a9d5501302116846a6f5db0634359b